UnityProject/Assets/Scripts/Managers/CargoManager.cs

```diff
@@ -486,6 +522,13 @@ public void AddBounty(ItemTrait trait, int amount, string description, int rewar
 			if(announce) CentComm.MakeAnnouncement(ChatTemplates.CentcomAnnounce, "A bounty for cargo has been issued from central communications", CentComm.UpdateSound.Notice);
 		}
 
+		public void AddBounty(CargoBounty bounty, bool announce)
+		{
+			if(bounty == null) return;
+			ActiveBounties.Add(bounty);
+			if(announce) CentComm.MakeAnnouncement(ChatTemplates.CentcomAnnounce, "A bounty for cargo has been issued from central communications", CentComm.UpdateSound.Notice);
```

Comment:
extract the announcement into a private method so it is only written once in the file.

Reply:
Done